mkdocs.1

```diff
@@ -0,0 +1,27 @@
+.TH "MKDOCS" "1" "0.1" "William Moreno Reyes" ""
+.SH "NAME"
+mkdocs \- Create HTML documentation from markdown sources
+.SH "SYNOPSIS"
+mkdocs [help|new|build|serve|gh\-deploy] {options}
+.SH "DESCRIPTION"
+MkDocs is a fast and simple way to create a website from source files written in Markdown, and configured with a YAML configuration file, the documentation  can be hosted anywhere, even in free hosting services like Read the Docs and GitHub Pages.
+.SH "OPTIONS"
+.TP
+\fBhelp\fP
+display this help and exit
+.TP
+\fBnew\fP 
+\fB<name>\fP create a new project with under the \fI<name>\fP directory
+.TP
+\fBbuild\fP
+make a HTML website from sources
+.TP
+\fBserve\fP
+start a server to view in real time the documentation
+.TP
+\fBgh\-deploy\fP
+commit changes to a GitHub Page
+.SH "AUTHOR"
+William Moreno Reyes
+.SH "COPYRIGHT"
+CC\-BY
```

Comment:
A license for the manpage itself makes no sense to me - wouldn't be comfortable with this pull request as it stands.